Kickstarter-iOS/Views/Cells/SettingsFollowCell.swift

```diff
@@ -63,6 +76,10 @@ internal final class SettingsFollowCell: UITableViewCell, ValueCell {
     self.followingSwitch.rac.on = self.viewModel.outputs.followingPrivacyOn
   }
 
+  func toggleOn(animated: Bool = true) {
```

Comment:
I don't feel strongly about the default attribute `animated: Bool = true` but I find it useful to always explicitly state this instead of calling something with `animated: true` - makes it easier to refactor later on